dcore/configuration.py

```diff
@@ -0,0 +1,81 @@
+"""
+Some simple tools for making model configuration nicer.
+"""
+
+
+class Configuration(object):
+    def __init__(self, **kwargs):
+
+        self.__setattr__ = self._proto__setattr__
+        for name, value in kwargs.iteritems():
+            print name, value
```

Comment:
I'm not sure what you mean when you say "the code" - the default parameters are there in the configuration class but can be overwritten when setup (e.g. in examples/dcmip_blah_blah_blah).  We should probably have the output going to a results directory whose name usually records some of the parameters used (e.g. dcmip_dx_dt) but it can't possibly record them all. I often use the same setup file to run with different parameters. I've used pickle to dump dictionaries of diagnostics - maybe we could dump a dictionary of parameters in the output directory?